README.md

```diff
@@ -176,6 +182,26 @@ An import will also cause dfe-analytics-dataform to correct occasions when there
 
 Unless ```enableMonitoring: false``` is set, dfe-analytics-dataform will automatically send a small packet of summary monitoring data to the ```monitoring.pipeline_snapshots``` table in the cross-service GCP project. This will include details of how many tables in your project have matching checksums, and how many rows are missing/extra in BigQuery when compared to the database.
 
+### Hidden fields
+```dfe-analytics-dataform``` is intended for use with the ```dfe-analytics``` Ruby gem. When used together they have the capability to use [BigQuery column-level access control](https://cloud.google.com/bigquery/docs/column-level-security-intro) to restrict access to certain 'hidden' fields. ```dfe-analytics``` is responsible for ensuring that hidden fields are streamed into the ```hidden_data``` field in your events table, while ```dfe-analytics-dataform``` is responsible for ensuring that data in that field remains hidden once it has been transformed into the other tables generated by dfe-analytics-dataform. If your Dataform pipeline transforms hidden data into fields in other tables which you also wish to hide, you are responsible for attaching policy tags to these fields. Google provide [documentation](https://cloud.google.com/dataform/docs/policy-tags) describing how to do this.
```

Comment:
Sometimes ```dfe-analytics-dataform``` is used sometimes it is just dfe-analytics-dataform (without ```) - I don;t know if you want to make this consistent

Reply:
Good spot. I've sorted this and for ```dfe-analytics``` and ```dfe-analytics-dotnet``` as well.